3711/1/parser/git_hub.rb

```diff
@@ -0,0 +1,78 @@
+require_relative '../ruby_gem'
+require 'json'
+require 'net/http'
+require 'nokogiri'
+require 'open-uri'
+
+module Parser
+  # :reek:TooManyInstanceVariables
+  class GitHub
+    def initialize(gem_name, source_url)
+      @url = source_url
+      @repo = @url[%r{\/[^.:\/]+\/[^.:\/]+}]
```

Comment:
I would move regex to constant but it would be better to get rid off regex at all

Reply:
Used `URI(@url).path` instead of RegExp